.github/workflows/gorgone.yml

```diff
@@ -11,6 +11,7 @@ on:
       - "centreon-gorgone/**"
       - "!centreon-gorgone/veracode.json"
       - "!centreon-gorgone/.veracode-exclusions"
+      - ".github/workflows/gorgone.yml"
```

Comment:
do not forget to remove this before merging, it is a good practice to not have the workflow file trigger runs outside of this pull request